www/about/policies/terms-of-service.md

```diff
@@ -0,0 +1,414 @@
+your legal guardian must consent to your use and must agree to be bound by
+these Terms.
+
+1. The Gratipay Model and General Rules
+
+  1. The Service is a platform to enable Teams of Gratipay Participants to
+     receive payments to fund Open Work. Open Work means that the Team provides
+a clear path for any individual to voluntarily begin contributing to the Team's
+work and to be paid for their contribution. Some examples of Teams performing
+Open Work include an open source software project that pays contributors, or a
+hackerspace that pays individuals to teach classes or manage its operations.
+
+  1. The Participant who establishes a Team is its Owner. Teams may optionally
+     have other Members.
+
+  1. When you establish a Team on the Service to fund your Open Work, you agree
+     that payments will be used exclusively to support the Open Work described
+in your Team's profile.
+
+  1. When you make a payment to a Team, it is a payment to fund the Open Work
+     the Team provides. You understand and acknowledge that your payment does
+not entitle you personally to receive any goods or services from the Team,
+unless the Team explicitly offers those things to Participants.
+
+  1. By accepting a payment from a Participant, a Team forms an agreement with
+     that Participant to perform the Open Work described in its profile. The
+Service is a platform to facilitate this agreement. Gratipay is not a party to
+the agreement, and is not responsible to either the Team or the Participant if
+either party breaches the terms of the agreement (e.g. if the Team fails to
+provide the promised services).
+
+  1. Gratipay does not provide refunds. Teams are responsible for producing the
+     Open Work they offer. Gratipay does not hold funds on behalf of Teams, and
+does not warrant or guarantee that Teams will provide the Open Work they
+promise.
+
+1. Payroll Rules
+
+  1. Teams can use the Service's Payroll feature to automatically divide
+     incoming payments among Team Members, according to the allocation set by
+the Members.
+
+  1. Gratipay does not determine how a Team's Payroll is allocated and is not
+     responsible for how Teams internally distribute payments. The agreement
+between Team Members and Owners regarding the allocation of payments is solely
+between those parties.
+
+1. Rules for Gratipay Teams
+
+  1. Teams may accept payments on the Service only for Open Work, meaning that
+     they provide a clear path for any individual to voluntarily begin
+contributing to the Team's work, and to be paid for their contribution.
+
+  1. A Team can only begin receiving payments from Participants once it has
+     provided a working withdrawal mechanism for receiving payments (e.g. bank
+account information).
+
+  1. Teams and their Open Work must be consistent with Gratipay's [Brand
+     Guidelines](http://inside.gratipay.com/big-picture/brand/), Community
+Guidelines, and Acceptable Use Policy.
```

Comment:
We don't have a Community Guidelines document. Seems like we should either write one or cut this out for now.

> [Brand Guidelines](http://inside.gratipay.com/big-picture/brand/)~~, Community Guidelines,~~ and Acceptable Use Policy.